### PR TITLE
Implement critical update card in `PatientTabScreen`

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/7223/implement-the-ui-for-critical-update-card-in-patienttabscreen

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt [hunk 1]
```diff
@@ -290,6 +291,17 @@ class PatientsTabScreen : BaseScreen<
...
   }
 
+  override fun openSimpleOnPlaystore() {
+    val packageManager = requireContext().packageManager
+    val intent = Intent(Intent.ACTION_VIEW, Uri.parse(PLAY_STORE_URL_FOR_SIMPLE))
+
+    if (intent.resolveActivity(packageManager) != null) {
+      requireContext().startActivity(intent)
+    } else {
+      CrashReporter.report(ActivityNotFoundException("Unable to play store url because no supporting apps were found."))
```
Comment (msasikanth) [inline app/src/main/java/org/simple/clinic/home/patients/PatientsTabScreen.kt:324]: It looks like we missed "open" from the error message. Can you add that, please?

```
Unable to open the play store URL...
```

File: app/src/main/res/layout/app_update_card.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (msasikanth) [inline app/src/main/res/layout/app_update_card.xml:2]: The design for this has a corner radius for this, should we wrap this with a card view or set a corner radius background?

File: app/src/main/res/layout/screen_patients.xml [hunk 3]
```diff
@@ -63,6 +63,10 @@
           android:id="@+id/simpleVideoLayout"
           layout="@layout/simple_video_layout" />
 
+        <include
```
Comment (msasikanth) [inline app/src/main/res/layout/screen_patients.xml:66]: I am guessing we will have a different PR for toggling between illustration, video, and update card?

**Review by msasikanth — CHANGES_REQUESTED** [review]
Overall this looks good. Left couple of comments PTAL.

File: app/src/main/res/layout/screen_patients.xml (hunk 3 above)
Comment (janhavisinghh) [reply app/src/main/res/layout/screen_patients.xml:66]: yess. That is a different story where everything comes together. [This](https://app.shortcut.com/simpledotorg/story/7236/implement-showing-app-update-nudges-based-on-the-priority-in-patienttabscreen) one.

**Comment by janhavisinghh** [comment]
@msasikanth I have added all the changes through interactive rebase as these were fairly small changes. Adding a final screenshot of the ui for reference. 
final

**Review by msasikanth — APPROVED** [review]

**Review by msasikanth — APPROVED** [review]

**Pull request merged**